content/fundamentals/unit-testing.md

```diff
@@ -339,7 +339,7 @@ providers: [
 
 > info **Hint** Change the `useClass` to `useExisting` to reference a registered provider instead of having Nest instantiate it behind the token.
 
-Now the `JwtAuthGuard` is visible to nest as a regular provider that can be overridden when creating the test module:
+Now the `JwtAuthGuard` is visible to nestjs as a regular provider that can be overridden when creating the `TestingModule`:
```

Comment:
```suggestion
Now the `JwtAuthGuard` is visible to nest as a regular provider that can be overridden when creating the `TestingModule`:
```

There's other places in the docs where the framework is referred to just as `nest` and it reads a bit more easily. I'd prefer to follow that if possible.